README.md

```diff
@@ -1,17 +1,139 @@
-# UCAN IPLD Schema
+# UCAN IPLD Schema v0.1.0
 
 ## Editors
 
-* _
+* [Irakli Gozalishvili](https://github.com/Gozala), DAG House
 
 ## Authors
 
-* _
+* [Hugo Dias](https://github.com/hugomrdias), DAG House
+* [Irakli Gozalishvili](https://github.com/Gozala), DAG House
+* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
 
 # 0 Abstract
 
-## Language
+[Interplanetary Linked Data (IPLD)](https://ipld.io/) is a consistent, highly general data model for content addressed linked data forming any DAG. This data model provides a convenient pivot between many serializations of the same information. Being a chained token model, UCANs can be very naturally expressed via IPLD. However, JWTs are not fully deterministic due to whitespace and key ordering. This specification defines an IPLD Schema for UCAN, and a canonical JWT serialization for compatibility with all other clients.
+
+# 1 Motivation
+
+The [core UCAN specification](https://github.com/ucan-wg/spec) is defined as a JWT to make it easily adoptable with existing tools, and as a common format for all implementations. There are many cases where different encodings are preferred for reasons such as compactness, machine-efficient formats, and so on. This specification outlines a format based on IPLD that can be deterministically encoded and decoded between many serialization formats, while still being able to encode as JWT for compatibility.
+
+# 2 IPLD Schema
+
+Unlike a JWT, the IPLD encoding of UCAN does not require separate header, claims, and signature fields.
+
+```ipldsch
+type UCAN struct {
+  ucv String 
+
+  iss DID
+  aud DID 
+  sig Signature
+
+  att [Capability] 
+  prf [&UCAN]
+  exp Int
+
+  fct [Fact]
+  nnc optional String
+  nbf optional Int
+} representation map {
+  field fct default []
+  field prf default []
+}
+```
+
+## 2.1 DID
+
+DIDs MUST be encoded as [DID](https://www.w3.org/TR/did-core/)s. The [`did:key` method](https://w3c-ccg.github.io/did-method-key/) is RECOMMENDED as it is self-contained.
+
+``` ipldsch
+type DID struct {
+  m String
+  i Bytes
+}
```

Comment:
This used to be `type DID = String`. I'm uncertain if perhaps `i Bytes` should actually be `i String`, but I'm happy either way.